### ISSUE TITLE
Parameter with named vector values does not work

### ISSUE THREAD

**Issue by AnaFialho** [madedotcom/retl#126]
It looks like parameters with named vector will cause problem with the query request and nothing is returned.

**Comment by AnaFialho** [comment]
Radhoc example works:

```r
leadtime <- bqExecuteFile(
  "ordering-tool/average-leadtime/extract-average-leadtime.sql", 
  product_states = unname(rmade::listProductStates(stock = TRUE)),
  priorities = c("A", "A*"),
  use.legacy.sql = FALSE
)
```

Radhoc example does not work:

```r
leadtime <- bqExecuteFile(
  "ordering-tool/average-leadtime/extract-average-leadtime.sql", 
  product_states = rmade::listProductStates(stock = TRUE),
  priorities = c("A", "A*"),
  use.legacy.sql = FALSE
)
```

**Comment by byapparov** [comment]
Fixed with byapparov/bigrquery#10 :

- commit hash: [5e7b5bc](https://github.com/byapparov/bigrquery/commit/5e7b5bc14849f0797e7bde3e43c7cb166a358b40)
